docs/runtime_options.md

```diff
@@ -194,6 +284,9 @@ interface that majority of users will rely on.
 * `instructions_retired` will be measured as non-grouped event and is specified using an advanced API that allows
 to specify any perf event available (some of them are not named and can't be specified with plain string). Event name 
 should be a human readable string that will become a metric name.
+* `cas_count_read` will be measured as uncore non-grouped event on all Integrated Memory Controllers Performance Monitoring Units because of unset `type` field and
+``uncore_imc`` prefix.
```

Comment:
Wrong formatting of ``uncore_imc`` ? It seems that single "`" will be enough here.